tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "moduleResolution": "node",
     "module": "es2020",
-    "lib": ["es2020"],
+    "lib": ["dom", "es2020"],
```

Comment:
ahh nice, I've got this included as part of my recover file system PR too for the same reason 😅

Reply:
Oh wait, I actually added it to the `email-recovery-template` repo i'm working on, so thanks for adding it here haha 🙏🏼 (working in too many template apps simultaneously 😅)

Reply:
Ah nice! Thanks for getting out the updates across all the templates. 🌊